### DISCUSSION TITLE
TURN server was configured without RFC 5780 support Error

### DISCUSSION THREAD

**Discussion by harrypotter033** [coturn/coturn#1607] [Q&A]
I am seeing continuous logs like this , please help  " error 420: Unknown attribute: TURN server was configured without RFC 5780 support"  I am using "coturn/coturn:4.6.2" docker image. Please help with this error 

`39483: (29): INFO: session 003000000000000011: realm <turn.development-somedomain.com> user <>: incoming packet OLD STUN message processed, error 420: Unknown attribute: TURN server was configured without RFC 5780 support

`

**Comment by eakraly** [comment]
Hi @harrypotter033 
Could you please post your config or arguments that you run turnserver with?
I think there are maybe 2 things in play:
1) your clients are trying to detect NAT behavior and sending CHANGE_REQUEST attribute (which turnserver rejects and logs this message)
2) someone is trying to use your server(s) for DDoS attack and crafting binding request which will result in maximum amplification (but fail because --no-rfc5780 is one of the anti-amplification attack mechanisms)

Check if the requests are coming from legit clients or not
Hope that helps you to find the issure
